rest/api_collections_test.go

```diff
@@ -269,6 +269,125 @@ func TestMultiCollectionDCP(t *testing.T) {
 	// require.NoError(t, rt.WaitForDoc(docID))
 }
 
+func TestMultiCollectionChannelAccess(t *testing.T) {
+	base.TestRequiresCollections(t)
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyAll)
+
+	if !base.TestUseXattrs() {
+		t.Skip("Test relies on import - needs xattrs")
+	}
```

Comment:
Same as other PR, there's nothing in this test that relies on import as far as I can tell

Reply:
addressed in [14b870c](https://github.com/couchbase/sync_gateway/pull/6012/commits/14b870cf30370886162344a52d2606ce581b002a)